README.md

```diff
@@ -40,6 +40,26 @@ API for the Trust Ethereum Wallet.
 * Run tests:
    ```$ npm run build && npm test```
 
+## Docker containers
+Install docker and docker-compose.
+
+Set in *~/.bashrc*
+```export COMPOSE_FILE=docker-compose.yml:docker-compose.dev.yml```
```

Comment:
is this required?

Reply:
If you want to mount volumes and expose ports - yes.
You can export COMPOSE_FILE in your terminal before using ./trust build and ./trust run.